src/lib/ens-helpers.ts

```diff
@@ -1,17 +1,16 @@
+import { bytesToPacket } from "@ensdomains/ensjs/utils";
 import { type Hex, concat, keccak256, namehash, toHex } from "viem";
 
-// TODO: pull from ens utils lib or something
 export const NAMEHASH_ZERO = namehash("");
```

Comment:
More specifically, this returns the id of the ENS Root. I usually give this a name such as `ENS_ROOT_NODE`.